src/services/item/plugins/publication/publicationState/fixtures.ts

```diff
@@ -0,0 +1,55 @@
+import { v4 } from 'uuid';
+
+import {
+  ItemTagType,
+  ItemType,
+  ItemValidationProcess,
+  ItemValidationStatus,
+  buildPathFromIds,
+} from '@graasp/sdk';
+
+import { ItemTag } from '../../itemTag/ItemTag';
+import { ItemMetadata, ItemValidationGroupStatus } from './types';
+
+export const ItemEntityFactory = (
```

Comment:
I'm not sure if it is what you're expecting by naming it as `...EntityFactory`, but it doesn't create a TypeORM Entity.
Otherwise, should we have this Factory in SDK ?

Reply:
Yeah you right ! It was initially for that, but it is not the case anymore... I'll rename it as `ItemMetadataFactory`. I don't think it's useful to put it into SDK, because it is only used to simplify the backend test...